src/get_clades_to_model.py

```diff
@@ -0,0 +1,75 @@
+"""
+Create a list of Sars-CoV-2 clades to model and saves it to the hub's auxiliary-data folder.
+
+This script wraps the virus-clade-utils package, which generates the clade list using the
+latest GenBank-based Sars-CoV-2 sequence metadata from Nextstrain.
+https://github.com/reichlab/virus-clade-utils
+
+Current parameters for the clade list:
+    threshold = .01,
+    threshold_weeks = 3 week
+    max_clades = 9
+
+The script is scheduled to run every Monday, for use in the modeling round that will open
+on the following Wednesday.
+
+To run the script manually:
+1. Install uv on your machine: https://docs.astral.sh/uv/getting-started/installation/
+2. From the root of this repo: uv run src/get_clades_to_model.py
+"""
+
+# /// script
```

Comment:
This script metadata instructs `uv run` to do its magic (create a transient Python virtual env with the requested dependencies and then run the script.

More info: https://docs.astral.sh/uv/guides/scripts/